docs/DOCUMENTATION_KO.md

```diff
@@ -0,0 +1,88 @@
+<h2>PBFT ����<h2/>
+
+<h4>�л� �ý��� ���</h4> 
+
+<ul>
+	
+<li> Fail-stop</li>
+	
+������ ���� �浹�������� �浹�� �ٸ� ������ �浹���� �߰߰���,��� return ����� �ʴ�.	
+<li> Byzantine fault</li>
+	
+Fail-stop ó�� ����� ���� �浹��������, �浹��� �� �ٸ� ������ �߰��ϱ� ���� �ʰ�, 
+  �߸������� �浹�� ����� return �Ѵ�.
+</ul>
+
+<h4>Byzantine ����?</h4>
+
+
+<p align="center"><img src="../images/pbft_img2.png" width="400px" height="200px"></p>
+
+
+
+<p>�켱 ��ü ��ũ��ũ�� ����� �� 3����� ���� �غ���� ��, �� �� commander����� �ϴ� ��尡 1�� ����ϰ� �ȴ�. 
+
+Commander ���� ��ǥ�� ������ ���� ����Ͽ� ������ Lieutenant ������ ��ǥ�� �� �� �ְ� �� ��, �� ��� �ݿ��Ͽ�
+ �ൿ� ����ϴ� ����� �Ѵ�. 
+�� 3���� ��尡 ��� �ùٸ� ����� ���ο��� ����Ѵٸ� 3���� ���� consensus�� ������ 
+�� �ְ� �ȴ�. 
+�׷��� 3���� ��� �� �ܺ��� ������� ���� ������ ����� ����ϰų� ������� �����ϴ� ��� �����ϴ� ����� 
+����ϴ� ��尡 ��� �� �ִ�.
+(Byzantine ���� ��� ��, 1���ϼ�, �� �̻��� �� �ִ�.) �� �� �� ��带 
+Byzantine ����� �Ѵ�. �� Byzantine ���� ���� ��� �߻��ϰ� �ȴ�. </p>
+
+<p>���� Byzantine ��尡 1����� ������ 2���� ����� 1���� ������ ����� �ùٸ������ ���� ���ްԵǾ� ȥ���� �Ͼ��
+ �ȴ�. �� �� ��� ���� �ùٸ� ������� �� ����� ��� ������ ��� �߻��Ѵ�. 
+
+��� ���ø� Ȯ���Ͽ� PBFT�� ���� �˾ƺ����� ����. </p>
+
+
+<h4>PBFT����</h4>
```

Comment:
I think just using <b>PBFT</b> is better. How about you?

Reply:
The reason I put the Byzantine node is, when I was studying the PBFT first time it was hard for me to understand the concept of  Byzantine node. Only using PBFT is a good way too, but I think this will help understand PBFT better.